src/app/Fake.DotNet.Cli/DotNet.fs

```diff
@@ -1156,18 +1165,18 @@ module DotNet =
         let argString = MSBuild.fromCliArguments msBuildArgs
 
         let binLogPath, args =
-            addBinaryLogger msBuildArgs.DisableInternalBinLog (args + " " + argString) common
+            addBinaryLogger msBuildArgs.DisableInternalBinLog (args @ argString) common
```

Comment:
Why space has been removed here?

Reply:
Before, ``args`` and ``argString`` were plain strings, which were just concated with a space between them.

Now, ``args`` and ``argString`` are both ``string list``, which get combined into a single list with the ``@`` operator.

The later code then uses either ``Args.toWindowsCommandLine`` (or ``CreateProcess.fromRawCommand``) to convert the string list into a single, properly quoted, string.

Reply:
with all that said I should also rename the variable from argString to argList